### PR TITLE
Fix: Update to match new Okta UI

### PR DESCRIPTION
**Pull request by adammarshallNR**
Per changes in https://github.com/newrelic/docs-website/blob/develop/src/content/docs/accounts/accounts/automated-user-management/okta-scimsso-application-configuration.mdx Also added step 7, to indicate where to add users/groups & added step 8, to make steps 6, 7 & 9 in the general SAML SSO instructions easier

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve? - **Okta UI has been changed.  Additionally, where to find required items in Okta for some of the general instructions are not intuitive.**
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc. - **I'm not sure where to add screenshot, but am happy to do so if required**
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more.mdx [hunk 1]
```diff
@@ -173,10 +173,12 @@ Before enabling SAML SSO using the instructions below, here are some things to u
 
        1. Go to [okta.com/](https://www.okta.com/) and sign in with an account that has administrator permissions.
        2. From the Okta home page, click on **Admin**.
-       3. From the Okta admin **Dashboard**, choose the **Applications** page. Click **Add Application**.
-       4. In the search field on the Okta **Add applications** page, enter "New Relic by organization" (not "New Relic by account") and then click on the application when it shows in the search results.
+       3. From the Okta admin **Dashboard**, choose the **Applications** page.
+       4. Click **Browse app catalog** and search for "New Relic by organization" (not "New Relic by account") and choose that from the results.
```
Comment (ally-sassman) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more.mdx:177]:
```suggestion
       4. Click **Browse app catalog** and search and select "New Relic by organization".
```

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more.mdx [hunk 2]
```diff
@@ -173,10 +173,12 @@ Before enabling SAML SSO using the instructions below, here are some things to u
...
        5. From the New Relic by Organization page, click on **Add**.
        6. From the Add New Relic by Organization page, check the two **Application visibility "Do not display..."** checkboxes and click on **Done**. We will make the application visible later after configuration is complete and provisioning has begun.
+       7. Open the newly created application in Okta and go to the **Assignments** tab. This is where you can add users and groups for Authentication.
```
Comment (ally-sassman) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more.mdx:180]:
```suggestion
       7. Open the newly created application in Okta and go to the **Assignments** tab. This is where you can add users and groups for authentication.
```

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more.mdx [hunk 3]
```diff
@@ -173,10 +173,12 @@ Before enabling SAML SSO using the instructions below, here are some things to u
...
+       8. From there, go to the **Sign On** tab. Under **Advanced Sign-on Settings**, you will see **Authentication Domain ID**, you will need to edit this field in step 9 of the general instructions below.  To the right you can click on **View SAML setup instructions**. Under step 7 of these instructions, you can find the URLs required for steps 6 & 7 in the general instructions below.
```
Comment (ally-sassman) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more.mdx:181]:
```suggestion
       8. From there, go to the **Sign On** tab. Under **Advanced Sign-on Settings**, you'll see **Authentication Domain ID**, which you'll need to edit this field in Step 9 of the general instructions below. To the right, click **View SAML setup instructions**. Under **Step 7** of these instructions, you can find the URLs required for Steps 6 & 7 in the general instructions below.
```

**Review by ghost — CHANGES_REQUESTED** [review]
See my suggestions

**Comment by adammarshallNR** [comment]
Thank you :) will try to remember for my next edit (or reread the style guide ;) )

**Review by ghost — DISMISSED** [review]

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more.mdx [hunk 4]
```diff
@@ -173,10 +173,12 @@ Before enabling SAML SSO using the instructions below, here are some things to u
 
        1. Go to [okta.com/](https://www.okta.com/) and sign in with an account that has administrator permissions.
        2. From the Okta home page, click on **Admin**.
-       3. From the Okta admin **Dashboard**, choose the **Applications** page. Click **Add Application**.
-       4. In the search field on the Okta **Add applications** page, enter "New Relic by organization" (not "New Relic by account") and then click on the application when it shows in the search results.
+       3. From the Okta admin **Dashboard**, choose the **Applications** page.
+       4. Click **Browse app catalog** and search and select "New Relic by organization".
```
Comment (ally-sassman) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more.mdx:177]:
```suggestion
       4. Click **Browse app catalog**, then search and select "New Relic by organization".
```

**Pull request merged**